packages/relay/src/formatters.ts

```diff
@@ -113,6 +113,19 @@ const parseNumericEnvVar = (envVarName: string, fallbackConstantKey: string): nu
     return value;
 }
 
+/**
+ * Parse value from hex string to number, if value lenght is bigger than tinybar to weibar coef, it means that it's in weibar, which needs conversion to weibar.
+ * @param value 
+ * @returns parsedValue
+ */
+const valueHexToInt = (value: string): number => {
+    if (BigInt(value).toString().length > constants.TINYBAR_TO_WEIBAR_COEF.toString().length) {
```

Comment:
Q: Won't value always be in weibar. Meaning we should always be applying the coefficient.

Reply:
Hm, actually you're right. Only ethereum tools use this, that means that we only get weibar values here. Fixed.